hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockInfoContiguous.java

```diff
@@ -28,12 +29,16 @@
 @InterfaceAudience.Private
 public class BlockInfoContiguous extends BlockInfo {
 
+  private boolean hasProvidedStorage;
+
   public BlockInfoContiguous(short size) {
     super(size);
+    hasProvidedStorage = false;
```

Comment:
Can we just set by default as false and not do it here?

Reply:
I removed it for now.